modular/gater/admin_handler.go

```diff
@@ -658,12 +662,22 @@ func (g *GateModular) replicateHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (g *GateModular) checkReplicatePermission(ctx context.Context, receiveTask gfsptask.GfSpReceivePieceTask, signatureAddr string) error {
-	objectInfo, err := g.baseApp.Consensus().QueryObjectInfo(ctx, receiveTask.ObjectInfo.BucketName, receiveTask.ObjectInfo.ObjectName)
+	var (
```

Comment:
please add UT.

You can mock the first and second response for the same call

Reply:
added